tmail-backend/pom.xml

```diff
@@ -236,647 +195,49 @@
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-ldap-guice</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-ldap-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-rabbitmq-guice</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-rabbitmq-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cli</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-api</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-ldap</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-memory</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-cassandra</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-configuration</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-common</artifactId>
+                <artifactId>james-project</artifactId>
                 <version>${james.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-common</artifactId>
-                <type>test-jar</type>
+                <artifactId>james-server-guice</artifactId>
```

Comment:
Why need to import the BOM of james-server-guice then?I think all submodules of james-server-guice are being declared in the james-project maven file no?

Reply:
THat is what I did.

Reply:
![Screenshot from 2021-05-31 10-57-24](https://user-images.githubusercontent.com/6928740/120137708-15b5f680-c1ff-11eb-8c2a-aa4e03827498.png)

Reply:
you did not really answered my question but ok I checked a bit myself more in details and indeed there is extra modules declared in james-server-guice...